packaging/os/pacman.py

```diff
@@ -233,14 +239,19 @@ def install_packages(module, pacman_path, state, packages, package_files):
 
     if install_c > 0:
         module.exit_json(changed=True, msg="installed %s package(s)" % (install_c))
+    
+    if install_c > 0:
+        module.exit_json(changed=True, msg="installed %s package(s)" % (install_c))
 
-    module.exit_json(changed=False, msg="package(s) already installed")
+    if state == 'latest' and len(package_err) > 0:
+        additionalmessage = "Install Latest not possible because Version cought not be fetched for: " + ''.join(package_err)
+    module.exit_json(changed=False, msg="package(s) already installed. %s" % (additionalmessage))
 
 
 def check_packages(module, pacman_path, packages, state):
     would_be_changed = []
     for package in packages:
-        installed, updated = query_package(module, pacman_path, package)
+        installed, updated, latestError = query_package(module, pacman_path, package)
```

Comment:
Would prefer the name just `unknown` instead of `latestError` for clarity and convention.